exonum/src/runtime/types.rs

```diff
@@ -63,12 +91,39 @@ impl AnyTx {
     }
 }
 
+/// The artifact identifier is required by the runtime to construct service instances.
+/// In other words an artifact identifier means same as class name, and a specific service
+/// instance is the class instance.
+///
+/// In string representation, the artifact identifier is written as follows:
+///
+/// `{runtime_id}:{artifact_name}`, where `runtime_id` is well known [runtime identifier],
```

Comment:
😕 but there is also version. Or can artifcat_name in turn be in format 'groupId:artifactId:version'?

Reply:
Artifact name may have its own naming conventions in different runtimes.